Add Computational Neuroscience

eselkin opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->


<!-- Please fill in the below placeholders -->

**https://github.com/eselkin/awesome-computational-neuroscience**

**The list identifies higher education researchers in computational neuroscience at a number of institutions around the world. It aims to capture all institutions that have a program offering in computational neuroscience, systems neuroscience, and theoretical neuroscience**


### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.

        - https://github.com/sindresorhus/awesome/pull/1577#issuecomment-512698789
        - https://github.com/sindresorhus/awesome/pull/1582#issuecomment-512695110 

- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).

Yes, but due to the geographical nature of the list it seemed that the multi-level contents was most appropriate. I could remove the institutions level but we'd still see one link that brought you to a massive list that couldn't be navigated around easily.

- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.

New category added "Higher Education" since nothing fit and it did not feel like a miscellaneous item.

- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.

Yes.

- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.

Waited.

- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).

The depth of the readme contents is the issue and the only issue that comes up with awesome-lint. Since it is a geographical list that hopes to chart all continents, I felt the depth was a necessary evil.

- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`

Yes. The first sentence describes the list and the second describes the mission.

- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**

This is a very time consuming process. It makes me appreciate the hard work other list creators have done.

- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`

Yes

- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`

Yes

- Non-generated Markdown file in a GitHub repo.
Yes... crafted by hand.

- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.

Yes

- Not a duplicate. Please search for existing submissions.

No duplicates, and no corresponding section

- Only has awesome items. Awesome lists are curations of the best, not everything.

The fact that all of these professors are from excellent universities with notable departments is key.

- Includes a project logo/illustration whenever possible.

        No logo

	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

Quotes thoroughly describe the goals of each of the researchers

- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).

Has awesome badge

	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.

Has Contents, is the first section
Since the geographic nature of the list has regions and the regions have schools, I could see no other way than nesting the list. If I kept it as one or two levels, the individual areas between the items would be far too large.

- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).

Has CC0

	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.

- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).

Has contributing.md with detailed description of contribution guidelines.

	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`

Diverges here considering the use of tables.

	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

No travis

- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

No inspiration

**Go to the top and read it again.**
done 2x

eselkin commented:
unicorn

AdrienTorris commented:
Hi, 

Great repo.

Your 'Contents' section has 3 levels of nested lists, only one is accepted.

> Has a Table of Contents section.


>     Should be named Contents, not Table of Contents.
>     Should be the first section in the list.
>     Should only have one level of nested lists, preferably none.

eselkin commented:
Due to the geographical nature of the list, I'm asking for a special exemption of this for this list (see the notes of the PR)

CompSciLauren commented:
Cool idea! Only things I would suggest are to change "provides" to "provide" in the first paragraph of the readme and fix the "U.S. West/Central/East" links in the contents section.

I like that the guide for contributing to your list is very descriptive.

As for the contents section having too many nested lists, I would agree with a special exception being made given the nature of the list. I don't see any good way to shorten it enough without making it difficult to navigate the list, especially as more locations are added over time.

eselkin commented:
Changed provides and fixed links in:
https://github.com/eselkin/awesome-computational-neuroscience/commit/31dfe5dcebb254584efeb89e617cb17073ef7776

sindresorhus commented:
It's weird that the researchers bios are linkified. I think their name should be linkified instead.

---

`^toc` = >`⬆️` or `^ Top`

Not everyone knows what `toc` means.

---

> 

Curtu, Rodica | Mathematics | ? | Google? | +
-- | -- | -- | -- | --

Don't include people if you don't have any info. Also the `?` link is invalid.

sindresorhus commented on readme.md:
```diff
@@ -517,6 +519,12 @@
+
+## Higher Education
+
+- [Computational Neuroscience](https://github.com/eselkin/awesome-computational-neuroscience) - Computational neuroscience institutions and researchers.
```
Couldn't this be in the `Computer Science` category?

eselkin replied:
In most schools it comes under the psychology, engineering, medicine or other departments. It is hardly ever in computer science.

sindresorhus replied:
Instead of repeating `Computational neuroscience` in the description, try to succinctly describe what `Computational neuroscience` is.

eselkin replied:
Done: https://github.com/sindresorhus/awesome/pull/1585/commits/b157203eb304c134536ec34cabd87fc94c4449a3

eselkin commented:
> It's weird that the researchers bios are linkified. I think their name should be linkified instead.
> 
> `^toc` = >`⬆️` or `^ Top`
> 
> Not everyone knows what `toc` means.
> 
> > 
> 
> Curtu, Rodica	Mathematics	?	Google?	+
> Don't include people if you don't have any info. Also the `?` link is invalid.

I'll take out the blanks and change the toc to the up arrow (I like that!)

eselkin commented:
Added the up-arrows instead of the ^toc, removed blank people

https://github.com/eselkin/awesome-computational-neuroscience/commit/17997ff84b5ea83617850c3bb00aa3e6a28e92ee
(but this accidentally screwed up formatting with the markdown autoformatter in VSCode) so...

https://github.com/eselkin/awesome-computational-neuroscience/commit/8ce5769f002e695c21f75d39e92c50be7d71f0ca

gwaygenomics commented on readme.md:
```diff
@@ -517,6 +519,12 @@
 - [Blockchain AI](https://github.com/steven2358/awesome-blockchain-ai#readme) - Blockchain projects for artificial intelligence and machine learning.
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 
+
```
Again two lines here?

eselkin replied:
If you see other sections, they have two lines.

gwaygenomics commented on readme.md:
```diff
@@ -507,6 +508,7 @@
 - [Network Analysis](https://github.com/briatte/awesome-network-analysis#readme)
 - [PCAPTools](https://github.com/caesar0301/awesome-pcaptools#readme)
 
+
```
Why adding an empty new line?

eselkin replied:
same as above.

eselkin commented:
> Comments on the specific list:
> 
> * Why is systems neuroscience in parentheses? Why not remove the parentheses?

Systems neuroscience is not a popular term, but occasionally used for a similar field.

> * This is a nice list! I can see it being very useful to computational neuroscience folks.

I hope so.

> * I am wondering if the continent -> country -> region breakdown is actually useful. Have users found it helpful? An alternative to the special request for increased depth can be accounted for by dropping country and region. This info can be added to the individual entries?

Yes, users found the list without regions to be unmanageably long.

sindresorhus commented:
It's not clear what `+/=/- computational` means.

sindresorhus commented:
> Yes. The first sentence describes the list and the second describes the mission.

It should also describe what `Computational Neuroscience` is. And `Computational Neuroscience` in that description should be linkified to https://en.wikipedia.org/wiki/Computational_neuroscience

eselkin commented:
> It's not clear what `+/=/- computational` means.

I was thinking of a way to explain this at the top. (+) refers to a researcher the university identifies as a computational neuroscientist, their bio consistently identifies a significant component of their research is in the field, and they have a significant body of work in the field. (=) refers to the fact that the university identifies them as practicing computational research and they have occasionally produced articles in the field. (-) means that the university identifies them as practicing computational neuroscience, their bio might also mention it, but I could not find articles of which they are an author that focus on the subject.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1163170622349991937
